content/en/docs/tasks/security/webhook/index.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Istio Webhook Management [Experimental]
+description: Shows how to manage webhooks in Istio through istioctl.
+weight: 100
+keywords: [security,webhook]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Istio has two webhooks: Galley and the sidecar injector. By default,
+these webhooks manage their own configurations. From a
+security perspective, this default behavior is not recommended because a compromised webhook could then conduct
+privilege escalation attacks.
+
+This task shows how to use `istioctl`, instead of the webhooks, to
+manage their configurations.
```

Comment:
would be good to add a short explanation here to indicate why is this approach better than the default before getting into detailed steps.

Reply:
@linsun We can explain the advantages of one approach over the other in a blog post. New readers of the task won't even know the old approach and having an explanation here will only confuse them.

Reply:
The explanation of why this approach is better is in the document as follows:
By default, these webhooks manage their own configurations. From a
security perspective, this default behavior is not recommended because a compromised webhook could then conduct privilege escalation attacks.